oShare/Chat Screen/ChatTableViewHandler.swift

```diff
@@ -65,13 +71,13 @@ class ChatTableViewHandler: NSObject, UITableViewDelegate, UITableViewDataSource
 			senderColor = UIColor.orange
 		}
 		
-		cell.textLabel?.font = UIFont.preferredFont(forTextStyle: .title3)
-		cell.textLabel?.text = message
-		cell.detailTextLabel?.font = UIFont.preferredFont(forTextStyle: .footnote)
-		cell.detailTextLabel?.text = senderLabelText
-		cell.detailTextLabel?.textColor = senderColor
+		cell?.textLabel?.font = UIFont.preferredFont(forTextStyle: .title3)
+		cell?.textLabel?.text = message
+		cell?.detailTextLabel?.font = UIFont.preferredFont(forTextStyle: .footnote)
+		cell?.detailTextLabel?.text = senderLabelText
+		cell?.detailTextLabel?.textColor = senderColor
 		
-		return cell
+		return cell!
```

Comment:
This was changed to enable testability. I wished to test the contents of the `detailTextLabel` which didn't exist when using just `register(UITableViewCell.self..` and dequeuing an ordinary cell.